src/lets_be_rational/black.rs

```diff
@@ -65,6 +69,79 @@ fn small_t_expansion_of_normalised_black_call(h: f64, t: f64) -> f64 {
     b.abs().max(0.0)
 }
 
+/// Computes the normalized Black call option value using a small-t expansion.
+///
+/// # Arguments
+///
+/// * `h` - The normalized log-moneyness (x / σ)
+/// * `t` - Half the total volatility (σ / 2)
+///
+/// # Returns
+///
+/// * `Some(f64)` containing the computed value if t < 0.21
+/// * `None` if t >= 0.21, indicating the approximation is not valid
+pub fn small_t_expansion_of_normalised_black_call(h: f64, t: f64) -> Option<f64> {
+    const T_THRESHOLD: f64 = 0.21;
+
+    if t >= T_THRESHOLD {
+        return None;
+    }
+
+    let t2 = t * t;
+    let h2 = h * h;
+
+    let a = 1.0 + h * (0.5 * SQRT_2PI) * erfcx(-FRAC_1_SQRT_2 * h);
```

Comment:
I noticed you reduced the YPrime function from Jackel's implementation, is there a reason for that?

Reply:
in Jackel's implementation we had 2 approaches
- original taylor implementation
- yprime

in solution i decide to implement default methods. In this situation it was just wrong condition ... 

both implementation are correct. Jeckel mentions yprime is better optimised so maybe i have to move it to yprime implementation instead taylor series

Reply:
I know why i dont use `Yprime` :)
in old implementation we had only taylor series:
https://github.com/hayden4r4/blackscholes-rust/blob/b6ed81b89fe70cffe24384b182234f385d92c776/lib/lets_be_rational.cpp#L256

I suggest to add `Yprime` in next issue. Then it will be similliar to Jackel implementation 2024